packages/rich-text/src/plugins/EmbeddedEntityInline/index.tsx

```diff
@@ -166,21 +162,10 @@ export function createEmbeddedEntityInlinePlugin(sdk): SlatePlugin {
           {
             type: INLINES.EMBEDDED_ENTRY,
             deserialize: (element) => {
-              const embeddedEntityInlineId = element.getAttribute('data-embedded-entity-inline-id');
-              if (!embeddedEntityInlineId) return;
-
-              return {
-                type: INLINES.EMBEDDED_ENTRY,
-                data: {
-                  target: {
-                    sys: {
-                      id: embeddedEntityInlineId,
-                      type: 'Link',
-                      linkType: 'Entry',
-                    },
-                  },
-                },
-              };
+              const entryId = element.getAttribute('data-embedded-entity-inline-id');
+              return entryId
+                ? createInlineEntryNode(entryId)
+                : undefined;
```

Comment:
Returning `undefined` can break other deserializations in my experience with Slate. Please, test pasting of other elements as well. I think SlateJS sees `undefined` as a value.

Reply:
The previous code
```
if (!embeddedEntityInlineId) return;
```
would also result in `undefined`, just that we're explicit about it now :thinking: